platform_versioned_docs/version-24.2/data_studios/index.mdx

```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
 
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application and the Seqera image template version itself. The tag string looks like this:
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
+```
+
+- `<tool_version>`: Analysis application version, such as `4.2.5` for JupyterLab
+- `<update_version>`: Seqera image template update integer version, such as `u1`, for instances where a breaking change is introduced
+- `<seqera_version>`: Seqera template image version, such as `0.7.0`
+
+Additionally, the Seqera version string looks like this:
+
+```ignore title="Seqera version tag subset"
+<major>.<minor>.<patch>
+```
+
+- `<major>`: Signifies breaking changes in the underlying Seqera container
+- `<minor>`: Signifies non-breaking changes in the underlying Seqera container
+- `<patch>`: Signifies minor non-breaking changes in the underlying Seqera container
```

Comment:
```suggestion
- `<patch>`: Signifies patch (non-breaking) changes in the underlying Seqera Connect client.
```

Reply:
It's a little confusing to call the `<minor>` version "non-breaking" and `<patch>` version "minor". Also, the minor version is actually a breaking change.